test/test_helpers/data_migrations.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
```

Comment:
TBH, we don't _need_ this file.
- Loading the migrations for testing could be done in https://github.com/3scale/porta/blob/master/test/test_helper.rb
- Making sure the `data_migrations` tables exists too
- `DataMigrations::DataMigrationTest` is not essential

The idea of putting it here was to set the convention somewhere. It's open to suggestions anyways.